rollup/jsPolyfills/_asyncNullishCoalesce.js

```diff
@@ -0,0 +1,4 @@
+export async function _asyncNullishCoalesce(lhs, rhsFn) {
+  // by checking for loose equality to `null`, we catch both `null` and `undefined`
+  return lhs != null ? lhs : rhsFn();
```

Comment:
It would be good to test these! We can do it later on though.

Reply:
Yes, though to the extent we test either sucrase or rollup at all, we have to trust these functions, as all but the interop ones are verbatim copied from what those packages inject. Still, if we had tests, you're right that we could try to refactor more of them to be more efficient.

(Not sure how much that's possible, but without tests I'm not even going to try to, for instance, refactor the optional chaining function. The interop ones I felt pretty confident about because they're super straightforward (as were most of my refactors, things like assigning directly rather than using `Object.defineProperty`), but for more complicated ones, I'd definitely want tests before I started mucking about.)

I'll throw it onto the backlog.